runtime/lua/vim/lsp/protocol.lua

```diff
@@ -730,7 +730,13 @@ function protocol.make_client_capabilities()
     experimental = nil;
     window = {
       workDoneProgress = true;
-    }
+      showMessage = {
+        messageActionItem = {
+          additionalPropertiesSupport = false;
+        };
+      };
+      showDocument = false;
```

Comment:
shouldn't it be {support = false} see https://microsoft.github.io/language-server-protocol/specifications/specification-current/#window_showDocument

Reply:
Yes